OurUmbraco/Our/Api/UpgradeCheckController.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web;
+using System.Web.Http;
+using System.Web.Security;
+using Newtonsoft.Json;
+using OurUmbraco.Our.Models;
+using OurUmbraco.Our.Services;
+using Umbraco.Core.Persistence;
+using Umbraco.Web.WebApi;
+
+namespace OurUmbraco.Our.Api
+{
+    public class UpgradeCheckController : UmbracoApiController
+    {
+        [HttpPost]
+        public IHttpActionResult Install(InstallationModel model)
```

Comment:
I think we should move this method to its own controller. Logically the two methods have nothing to do with each other

Reply:
You are right! Will refactor 👌